appveyor.yml

```diff
@@ -36,6 +36,7 @@ install:
     - "powershell ci-helpers/appveyor/install-miniconda.ps1"
     - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
     - "activate test"
+    - "conda install -c conda-forge pytest-qt"
```

Comment:
Please remove this line and instead add below `CONDA_DEPENDENCIES` this line

`PIP_DEPENDENCIES: "pytest-qt"`